datafusion/physical-plan/src/aggregates/mod.rs

```diff
@@ -283,10 +283,6 @@ pub struct AggregateExec {
     pub input: Arc<dyn ExecutionPlan>,
     /// Schema after the aggregate is applied
     schema: SchemaRef,
-    /// Input schema before any aggregation is applied. For partial aggregate this will be the
```

Comment:
This PR proposes deleting this field as it is redundant

Reply:
It appears to have been added by @andygrove  in https://github.com/apache/arrow-datafusion/commit/b71e3ee5b7b85cea339d117e45731c5f9c7862db / https://github.com/apache/arrow/pull/9481

Since DataFusion has its own Serde of physical plans, I don't think we need this anymore